documentation/manual/src/main/asciidoc/en-US/modules/configuration.asciidoc

```diff
@@ -36,7 +36,7 @@ The provider name is [classname]`org.hibernate.ogm.jpa.HibernateOgmPersistence`.
             <property name="hibernate.transaction.jta.platform"
                       value="JBossTS" />
             <property name="hibernate.ogm.datastore.provider"
-                      value="infinispan" />
+                      value="infinispan_embedded" />
```

Comment:
Running `grep -R '"infinispan"'` returned several remaining occurrences of the deprecated string in the tests. Is that on purpose?